src/DynamoCore/Models/AnnotationModel.cs

```diff
@@ -0,0 +1,379 @@
+﻿
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Runtime.Remoting.Messaging;
+using System.Xml;
+using Dynamo.Properties;
+using Dynamo.Utilities;
+using System.Collections.Generic;
+using ICSharpCode.AvalonEdit.Editing;
+using ProtoCore.AST;
+
+namespace Dynamo.Models
+{
+    public class AnnotationModel : ModelBase
+    {
+        #region Properties
+        public double InitialTop { get; set; } //required to calculate the TOP position in a group         
+        public double InitialHeight { get; set; } //required to calculate the HEIGHT of a group 
+        private string modelGuids { get; set; }        
+        private string _text;
+        public string Text
+        {
+            get { return _text; }
+            set
```

Comment:
All private variables should be lowercase.  Yes, the code base is inconsistent about this, but this matches .NET coding standards.